lib/puma/binder.rb

```diff
@@ -374,7 +381,7 @@ def add_unix_listener(path, umask=nil, mode=nil, backlog=1024)
           end
         end
 
-        s = UNIXServer.new(path)
+        s = UNIXServer.new path.sub(/\A@/, "\0")
```

Comment:
Everywhere that we check for a socket starting with `@`, I think we should have a comment explaining this a check for an abstract socket.

Reply:
Done